roles/adw/tasks/main.yml

```diff
@@ -14,7 +14,7 @@
   get_url:
     url: "{{ downloads.adw_zip_url }}"
     dest: "{{ download_location }}/alfresco-digital-workspace-{{ adw.version }}.zip"
-    checksum: "sha1:{{ downloads.adw_zip_sha1_checksum }}"
+    checksum: "sha1:{{ lookup('url', downloads.adw_zip_sha1_checksum_url, username=nexus_user, password=nexus_password) }}"
```

Comment:
This approach requires to default-set both `nexus_username` & `nexus_password` for people who deploy community and have no credentials to provide.
I think this would better fit in `roles/common/defaults/main.yml` (instead of currently used ugly string tpl in `roles/common/vars/main.yml)